README.md

```diff
@@ -151,3 +151,14 @@ Turns preview into a simple console for testing out ES6 code. Use `console.log()
   es6Console={true}
   codeText={es6Example} />
 ```
+
+###babelConfig
+_React.PropTypes.object_
+
+An object of [configuration options](http://babeljs.io/docs/usage/options/) that the playground should pass to babel. Use to adjust, or limit the features you want the playground to support.
```

Comment:
<!-- Go to www.addthis.com/dashboard to customize your tools -->
<script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-55660b4008d9c380" async="async"></script>